frontend/pages/admin/TeamManagementPage/TeamDetailsWrapper/TeamDetailsWrapper.tsx

```diff
@@ -131,6 +131,7 @@ const TeamDetailsWrapper = ({
   }>({});
 
   const { refetch: refetchMe } = useQuery(["me"], () => usersAPI.me(), {
+    enabled: false,
```

Comment:
This is how we'll handle instances where we want to be able to hit the `/me` endpoint after the route loads based on something that happens in the route.